lightkurve/targetpixelfile.py

```diff
@@ -792,7 +792,19 @@ def _aperture_photometry(self, aperture_mask, centroid_method='moments'):
         centroid_col, centroid_row = self.estimate_centroids(apmask, method=centroid_method)
 
         # Estimate flux
-        flux = np.nansum(self.flux[:, apmask], axis=1)
+        if flux_method == "sum":
+            flux = np.nansum(self.flux[:, apmask], axis=1)
+            
+        elif flux_method == "median":
+            flux = np.nanmedian(self.flux[:, apmask], axis=1)
+            
+        elif flux_method == "mean":
+            flux = np.nanmean(self.flux[:, apmask], axis=1)
+            
+        #Want default flux to be sum if not specified
+        #In the future we may wish to add a user specified function 
+        else:
+            flux = np.nansum(self.flux[:, apmask], axis=1)
```

Comment:
If any other value is passed to `flux_method`, the user is using the function incorrectly. We should let them know by raising an error.  I recommend doing so by changing line 807 into this:
```suggestion
            raise ValueError("`flux_method` must be one of 'sum', 'median', or 'mean'.")
```

Reply:
Done